### PR TITLE
fix(developer): language history was out of date

### LINKED ISSUE #118: Keyman Language History is not updated with 11.0, 12.0, 13.0 changes:
* Keyman Language History is not updated with 11.0, 12.0, 13.0 changes:
  developer\language\guide\history.php

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #118.

### CODE REVIEW

File: developer/language/guide/history.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Keyman 4
+
+*   The `+` character became required, and usage became stricter to follow the standard now: the plus character must separate
+    context from keystroke in a rule.
+*   Keyboards are now compiled by the developer into .kmx files, rather than being distributed as .kmn files.
+
+## Keyman 5
+
+*   Added [Unicode support](unicode)
+*   Added [Input Method eXtension support](../reference/call)
```
Comment (darcywong00) [inline developer/language/guide/history.md]:
```suggestion
*   Added [Input Method Extension support](../reference/call)
 ```

Comment (mcdurdin) [reply developer/language/guide/history.md]: Sorry ... but the term in Keyman is IMX to disambiguate from system IMEs :)

File: developer/language/guide/history.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+*   Added [`reset()` statement](../reference/reset)
+
+## Keyman 9
+
+*   Added [touch device support](/developer/current-version/guides/develop/)
+*   Added [`&keyboardversion` store](../reference/keyboardversion)
+*   Added [`&kmw_embedcss` store](../reference/kmw_embedcss)
+*   Added [`platform` statement](../reference/platform)
+*   Added [`layer` statement](../reference/layer)
+*   Added [`&targets` statement](../reference/targets)
```
Comment (darcywong00) [inline developer/language/guide/history.md]:
```suggestion
*   Added [`&targets` store](../reference/targets)
```

File: developer/language/guide/history.md (hunk 1 above)
Comment (darcywong00) [reply developer/language/guide/history.md]: ah, then the Description on https://help.keyman.com/developer/language/reference/call should be consistent
> The call() statement calls an Input Method Extension (IMX) DLL.

Comment (mcdurdin) [reply developer/language/guide/history.md]: 😁😁😁. Consistency is for other people!

**Pull request merged**